### PR TITLE
Episode 14

### PR DESCRIPTION
**Pull request by sockeqwe**
Unfortunately the regular feed validator doesn't seem to operate.
http://castfeedvalidator.com)

https://raw.githubusercontent.com/artem-zinnatullin/TheContext-Podcast/Episode_14/feed.rss

I have used this one:
https://podba.se/validate/?url=https://raw.githubusercontent.com/artem-zinnatullin/TheContext-Podcast/Episode_14/feed.rss

TODO
- [x] Rename mp3 file to **https://artemzin.com/static/thecontext/episodes/The.Context.episode.14.mp3**
- [x] Upload mp3 file
- [ ] add git tag
- [ ] add to release section on github

### CODE REVIEW

File: feed.rss [hunk 1]
```diff
@@ -963,5 +963,62 @@
 			]]>
 		</itunes:summary>
 	</item>
+	<item>
+		<guid>https://artemzin.com/static/thecontext/episodes/The.Context.episode.14.mp3</guid>
+		<title>Episode 14: RxJava — the Present and Future with David Karnok (core maintainer)</title>
+		<link>https://github.com/artem-zinnatullin/TheContext-Podcast/blob/master/show_notes/Episode_14.md</link>
+		<atom:link rel="replies" type="text/html" href="https://github.com/artem-zinnatullin/TheContext-Podcast/issues/80"/>
+		<itunes:duration>01:21:11</itunes:duration>
+		<pubDate>Thu, 5 Oktober 2017 09:30:00 EST</pubDate>
```
Comment (artem-zinnatullin) [inline feed.rss]: Oktober lol

File: feed.rss [hunk 2]
```diff
@@ -963,5 +963,62 @@
...
+		<enclosure url="https://artemzin.com/static/thecontext/episodes/The.Context.episode.14.mp3" length="77973101" type="audio/mpeg"/>
+		<atom:contributor>
+			<atom:name>David Karnok</atom:name>
+			<atom:uri>https://twitter.com/eric_kuck</atom:uri>
```
Comment (artem-zinnatullin) [inline feed.rss]: `https://twitter.com/akarnokd`

File: feed.rss [hunk 3]
```diff
@@ -963,5 +963,62 @@
...
+			<![CDATA[
+			<h1>Episode 14: RxJava — the Present and Future with Dávid Karnok (core maintainer)</h1>
+
+			<h3><a href="https://github.com/artem-zinnatullin/TheContext-Podcast">How to listen &amp; subscribe</a></h3>
+
+			<ul>
+			<li><a href="https://github.com/artem-zinnatullin/TheContext-Podcast/issues/80">Discussion after the episode</a></li>
+			</ul>
+
+			<p>In this episode Artem and Hannes chatted with Dávid Karnok about present and future of <a href="https://github.com/ReactiveX/RxJava">RxJava's</a>. We discussed:</p>
```
Comment (artem-zinnatullin) [inline feed.rss]: `David`, `a` instead of `á`

File: feed.rss [hunk 4]
```diff
@@ -963,5 +963,62 @@
...
+			<li><a href="https://github.com/JakeWharton/Reagent/">Reagent</a>: Experiments for future reactive libraries by using Kotlin Coroutines by JakeWharton</li>
+			<li><a href="http://akarnokd.blogspot.de/2017/09/rxjava-vs-kotlin-coroutines-quick-look.html">RxJava vs. Kotlin Coroutines, a quick look</a> by Dávid Karnok
+			<ul><li><a href="http://akarnokd.blogspot.de/2017/09/rewriting-rxjava-with-kotlin-coroutines.html">Rewriting RxJava with Kotlin Coroutines?</a> by Dávid Karnok</li></ul></li>
+			<li><a href="http://akarnokd.blogspot.de/2017/09/interoperation-between-rxjava-and.html">Interoperation between RxJava and Kotlin Coroutines</a> by Dávid Karnok</li>
+			</ul>
+
+			<h4>Guest:</h4>
+
+			<ul>
+			<li><p>Dávid Karnok <a href="https://twitter.com/akarnokd">@akarnokd</a>, <a href="https://github.com/akarnokd">GitHub</a>, <a href="http://akarnokd.blogspot.de">Blog</a></p>
```
Comment (artem-zinnatullin) [inline feed.rss]: `a`

File: feed.rss [hunk 5]
```diff
@@ -963,5 +963,62 @@
...
+			<h4>Hosts:</h4>
+
+			<ul><li>Artem Zinnatullin <a href="https://twitter.com/artem_zin">@artem_zin</a>, <a href="http://artemzin.com">personal blog</a>, <a href="https://github.com/artem-zinnatullin">GitHub</a></li>
+			<li>Hannes Dorfmann <a href="https://twitter.com/sockeqwe">@sockeqwe</a>, <a href="http://hannesdorfmann.com">personal blog</a>, <a href="https://github.com/sockeqwe">GitHub</a></li></ul></li>
+			</ul>
+
+			<h4>Additional links:</h4>
+
+			<ul>
+			<li><a href="https://github.com/artem-zinnatullin/TheContext-Podcast/blob/master/show_notes/Episode_3_Part_1.md">Episode 3 of The Context Podcast featuren Dávid Karnok</a></li>
```
Comment (artem-zinnatullin) [inline feed.rss]: `a`

File: show_notes/Episode_14.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
+# Episode 14: RxJava — the Present and Future with Dávid Karnok (core maintainer)
+
+### [How to listen & subscribe](https://github.com/artem-zinnatullin/TheContext-Podcast)
+
+ - [Discussion after the episode](https://github.com/artem-zinnatullin/TheContext-Podcast/issues/80)
+
+In this episode Artem and Hannes chatted with Dávid Karnok about present and future of [RxJava's](https://github.com/ReactiveX/RxJava). We discussed:
```
Comment (artem-zinnatullin) [inline show_notes/Episode_14.md]: `a`

File: show_notes/Episode_14.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+ - Artem Zinnatullin's [Reply to Kaushik Gopal's aricle: "RxJava 1 -> RxJava 2 (Understanding the Changes)"](https://artemzin.com/blog/reply-to-kaushik-gopals-aricle-rxjava-1-rxjava-2-understanding-the-changes/)
+ - [Kotlin Coroutines](https://kotlinlang.org/docs/reference/coroutines.html)
+ - [Reagent](https://github.com/JakeWharton/Reagent/): Experiments for future reactive libraries by using Kotlin Coroutines by JakeWharton
+ - [RxJava vs. Kotlin Coroutines, a quick look](http://akarnokd.blogspot.de/2017/09/rxjava-vs-kotlin-coroutines-quick-look.html) by Dávid Karnok
+  - [Rewriting RxJava with Kotlin Coroutines?](http://akarnokd.blogspot.de/2017/09/rewriting-rxjava-with-kotlin-coroutines.html) by Dávid Karnok
+ - [Interoperation between RxJava and Kotlin Coroutines](http://akarnokd.blogspot.de/2017/09/interoperation-between-rxjava-and.html) by Dávid Karnok
+
+#### Guest:
+
+ - Dávid Karnok [@akarnokd](https://twitter.com/akarnokd), [GitHub](https://github.com/akarnokd), [Blog](http://akarnokd.blogspot.de)
```
Comment (artem-zinnatullin) [inline show_notes/Episode_14.md]: `a`

File: feed.rss (hunk 3 above)
Comment (artem-zinnatullin) [reply feed.rss]: Otherwise it could be hard to use "Find on Page" and google

**Comment by sockeqwe** [comment]
:facepalm: could you change that, Im boarding now, wont be online until
tomorrow afternoon ... Need some sleep

**Comment by artem-zinnatullin** [comment]
I'll take care of that, don't worry :)

**Comment by sockeqwe** [comment]
You are awesome dude!

File: feed.rss (hunk 1 above)
Comment (vanniktech) [reply feed.rss]: Germans will take over ;)

File: README.md [hunk 8]
```diff
@@ -32,6 +32,10 @@ Hosts: Artem Zinnatullin [@artem_zin](https://twitter.com/artem_zin), Hannes Dor
 
 ### [Episodes](https://github.com/artem-zinnatullin/TheContext-Podcast/releases)
 
+##### Episode 14: RxJava — the Present and Future with [David Karnok](https://twitter.com/akarnokd) (core maintainer)
```
Comment (artem-zinnatullin) [inline README.md]: Ah, "the Present and the Future"

**Pull request merged**